btc/src/main/kotlin/cli/GenerateBlocks.kt

```diff
@@ -0,0 +1,45 @@
+@file:JvmName("BtcGenerateBlocksMain")
+
+package cli
+
+import com.github.jleskovar.btcrpc.BitcoinRpcClientFactory
+import com.github.kittinunf.result.Result
+import config.loadConfigs
+import mu.KLogging
+
+private val logger = KLogging().logger
+
+private val btcNodeRpcConfig =
+    loadConfigs("btc-node-rpc", BtcNodeRpcConfig::class.java, "/btc/node-rpc.properties").get()
+
+private val rpcClient = BitcoinRpcClientFactory.createClient(
+    user = btcNodeRpcConfig.user,
+    password = btcNodeRpcConfig.password,
+    host = btcNodeRpcConfig.host,
+    port = btcNodeRpcConfig.port,
+    secure = false
+)
+
+/**
+ * Generates blocks in Bitcoin blockchain
+ * @param args - array full of arguments. args[0] - number of blocks to generate
+ */
+fun main(args: Array<String>) {
+    val blocksToGenerate = args[0].toInt()
```

Comment:
I suggest to add some default value (6) if an argument value is not provided or it's less than 1